eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/GA4GHBeaconResponse.java

```diff
@@ -0,0 +1,83 @@
+package uk.ac.ebi.eva.server.ws.ga4gh;
+
+public class GA4GHBeaconResponse {
+
+    private String chromosome;
+
+    private Integer start;
+
+    private String allele;
+
+    private String datasetIds;
+
+    private boolean exists;
+
+    private String errorMessage;
+
+    public GA4GHBeaconResponse() {
```

Comment:
Empty constructor is required, but if I recall correctly class protected is enough.